### ISSUE TITLE
JS version of code examples 

### ISSUE THREAD

**Issue by abe33** [atom/docs#36]
What do you think of allowing the coffe code examples in the docs to be compiled on the fly to JS? 

I did that once for [a pet project](http://abe33.github.io/spectacular/) and found that it was quite useful.

I think the following feature would also be needed: 

- A toggle in the interface so that the user can select the output it prefer for the whole docs, the setting would be stored in a cookie and used on every pages to force the examples compilation on page load.
- As the JS output is not always the most beautiful one, maybe it could be useful to have a way to specify a JS example along the coffee one that will be used instead of compiling the coffee code.

**Comment by thomasjo** [comment]
The JS examples would really need to be handwritten for them to have any real benefit. The result of running code through the CoffeeScript compiler is not really meant for human consumption.

**TL;DR** Not opposed to providing JS examples, but only as long as they are handcrafted.

**Comment by dcarral** [comment]
@thomasjo Just came across this issue looking for open ones regarding this CoffeeScript/JavaScript topic. Given the lapsed time, I was wondering if nowadays you folks have any ideas to work on something related.

**Comment by 50Wliu** [comment]
@dcarral we're actively moving away from Coffeescript and as part of that transition are slowly updating our documentation to use JavaScript instead of Coffeescript.  See #377 for one example.

**Comment by liuderchi** [comment]
@50Wliu I have read #377 then found a context paragraph which may be inconsistent with it.

[Chapter 3: Tools of the Trade](http://flight-manual.atom.io/hacking-atom/sections/tools-of-the-trade/)

> ..., Since all of Atom is implemented using web technologies, we have to assume you know web technologies such as JavaScript and CSS. **Specifically, we'll be implementing everything in CoffeeScript and Less**, which are preprocessors for Javascript and CSS respectively.

Would that be another issue since we now can use JS?
